src/main.rs

```diff
@@ -85,6 +87,41 @@ fn run_script(script_file: &String) {
     }
 }
 
+fn is_none(objref: PyObjectRef) -> bool {
```

Comment:
For now, this is okay, but None should be a singleton. When we have that, we can use the singleton and check with the _is function of the vm.